enzy_htp/_config/mole2_config.py

```diff
@@ -40,7 +40,7 @@ class Mole2Config(BaseConfig):
     MONO:str="mono"
     """Path to mono as a str()."""
 
-    MOLE2:str="~/Downloads/Mole2_plugin/MOLE25_binary/mole2.exe"
```

Comment:
Make this an absolute path on ACCRE and only the permission for all yang_lab members